fuse_examples/tutorials/hello_world/hello_world.ipynb

```diff
@@ -0,0 +1,1411 @@
+       "9999  [4.668895e-06, 1.5146396e-08, 2.947092e-05, 4....           6  \n",
+       "\n",
+       "[10000 rows x 4 columns]"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "validation_dataloader = DataLoader(dataset=validation_dataset, collate_fn=validation_dataset.collate_fn, batch_size=2, num_workers=2)\n",
+    "\n",
+    "## Manager for inference\n",
+    "manager = FuseManagerDefault()\n",
+    "output_columns = ['model.output.classification', 'data.label']\n",
+    "manager.infer(data_loader=validation_dataloader,\n",
+    "                input_model_dir=paths['model_dir'],\n",
+    "                checkpoint=infer_common_params['checkpoint'],\n",
+    "                output_columns=output_columns,\n",
+    "                output_file_name=os.path.join(paths[\"inference_dir\"], infer_common_params[\"infer_filename\"]))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## **Evaluation**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### **Define EVAL Common Params**\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "EVAL_COMMON_PARAMS = {}\n",
+    "EVAL_COMMON_PARAMS['infer_filename'] = INFER_COMMON_PARAMS['infer_filename']\n",
+    "eval_common_params = EVAL_COMMON_PARAMS"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### **Calculate metrics**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class_names = [str(i) for i in range(10)]\n",
+    "\n",
+    "# metrics\n",
+    "metrics = OrderedDict([\n",
+    "    ('operation_point', MetricApplyThresholds(pred='model.output.classification')), # will apply argmax\n",
+    "    ('accuracy', MetricAccuracy(pred='results:metrics.operation_point.cls_pred', target='data.label')),\n",
+    "    ('roc', MetricROCCurve(pred='model.output.classification', target='data.label', class_names=class_names, output_filename=os.path.join(paths['inference_dir'], 'roc_curve.png'))),\n",
+    "    ('auc', MetricAUCROC(pred='model.output.classification', target='data.label', class_names=class_names)),\n",
+    "])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### **Evaluate**\n",
+    "Using the Evaluator from the evaluation package of FuseMedML (fuse.eval) which is a standalone library for evaluating ML models that not necessarily trained with FuseMedML.\n",
+    "\n",
+    "More details and examples for the evaluation package can be found [here](https://github.com/IBM/fuse-med-ml/blob/master/fuse/eval/README.md)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
```

Comment:
remove the long output

Reply:
I've used io's walkaround to prevent it from printing.. do you have a nittier way?